### ISSUE TITLE
Reevaluate how we ascertain and represent candidates' classes

### ISSUE THREAD

**Issue by arynh** [hack4impact-uiuc/h4i-recruitment#353]
Currently we ask for graduation date on the initial form, though this often results in some less than accurate representations of how long people might stay at UIUC. (@arpanlaha's "fake" label helps, but it would be nice if we could implement @arpanlaha as software.) Some possible solutions to this problem include:  allow portal users to modify the class of candidates. Through personal connections or graduation date or even information gathered from the candidate's resume, we can usually make a more accurate assessment of the class of the candidate. We can also change the pipeline at the very beginning; ask applicants for their entry semester.

**Comment by josh-byster** [comment]
Definitely, thanks @arynh. Another Idea I was thinking which is related is that we could have them note both their start and end date down (so have like two options—December and May and then a year dropdown). This way, we can filter however we like—if we wanted to know class by graduation date, we could group by that. If we wanted to know class by how many years they have been at UIUC, we could filter down by that. And I think it would be good to also have manual override like you said, Aryn, because for someone that transferred, we'd want to take that into account and other special circumstances.
